src/application/F3DStarter.cxx

```diff
@@ -108,37 +109,40 @@ int F3DStarter::Start(int argc, char** argv)
     this->Internals->Engine->getWindow().setIcon(F3DIcon, sizeof(F3DIcon));
   }
 
+  // Copy command line options into the actual engine options
+  f3d::options& options = this->Internals->Engine->getOptions();
+  options = commandLineOptions;
```

Comment:
Instead of copying the `f3d::options` class, why not using the reference directly and pass it to `GetOptionsFromCommandLine`?  
I don't like the idea of making this class copyable.

Reply:
you need to read the options to be able to create the engine, and you need the engine to create the options.